src/VoyagerServiceProvider.php

```diff
@@ -117,7 +117,11 @@ protected function registerViewComposers()
      */
     protected function addStorageSymlinkAlert()
     {
-        $currentRouteAction = app('router')->current()->getAction();
+        if (app('router')->current() !== null) {
+            $currentRouteAction = app('router')->current()->getAction();
```

Comment:
Yeah, I wouldn't create a new variable for that, since its already stored in `$this->app->router` (counts as a variable, stored in memory).

So it'd be

```
if ($this->app->router->current() !== null) {
    $currentRouteAction = $this->app->router->current()->getAction();
} else  {
    $currentRouteAction = null;
}
```